allensdk/test/brain_observatory/vbn_2020/metadata_writer/conftest.py

```diff
@@ -0,0 +1,110 @@
+import pytest
```

Comment:
I think you meant to call this package `vbn_2022`

Reply:
....I've been making that typo all week....